src/main/frontend/src/components/landingpage/Categories.js

```diff
@@ -0,0 +1,43 @@
+import classes from './Categories.module.css';
+
+function Categories(){
+    return(
+        <section className={classes.categorieslist}>
+            <div className={`${classes.categoryitem} ${classes.categoryitemheading}`}>
```

Comment:
Instead of listing each category like this, this component should receive categories as prop, so we could have smtg like: 
```suggestion
            function Categories({ categories }){
    return(
        <section className={classes.categorieslist}>
        {
           categories.map(category => {
              <div className={classes.categoryitem}>
                <h3 className={`${classes.categoriesheading} ${classes.categoryitemtext}`}>{category.name}</h3>
            </div>
           })
        }
        </section>
```
Until you have controller for fetching categories, you can define categories array in Home.js and pass it here